asynkio/src/test/java/me/nikhilchaudhari/asynkioapp/NetworkCallTest.java

```diff
@@ -0,0 +1,68 @@
+package me.nikhilchaudhari.asynkioapp;
+
+import org.junit.Test;
+
+import java.io.InputStream;
+import java.nio.file.Files;
+import java.nio.file.Paths;
+import java.security.KeyStore;
+import java.util.Collections;
+
+import javax.net.ssl.KeyManager;
+import javax.net.ssl.KeyManagerFactory;
+import javax.net.ssl.SSLContext;
+
+import me.nikhilchaudhari.asynkio.core.NetworkCall;
+import me.nikhilchaudhari.asynkio.helper.RawFiles;
+import me.nikhilchaudhari.asynkio.response.Response;
+
+import static org.hamcrest.CoreMatchers.is;
+import static org.junit.Assert.assertThat;
+
+public class NetworkCallTest {
```

Comment:
Would you mind writing this in kotlin. 
you can add kotlin support for test folder as well in gradle

Reply:
Sure, just changed it to kotlin.
But what do you mean exactly with:
> you can add kotlin support for test folder as well in gradle

Reply:
Thanks a lot.
I thought kotlin dependency is not added for the test folder 🤦 . 
but it already there, sorry my bad ✋.